### ISSUE TITLE
cy8ckit_028_tft_init() is required prior to GUI_Init()

### ISSUE THREAD

**Issue by noritan** [cypresssemiconductorco/CypressAcademy_MTB101_Files#2]
**Prerequisites**

Can you reproduce the problem reliably?
I can reproduce this on my CY8CKIT-062-WiFi-BT and CY8CKIT-028-TFT

Did you check current release notes for known issues?

If this is not the latest release, have you checked newer releases? 

**Description**:

Following the "Exercise 8: Write a message to the TFT shield" on page 63 of the document "MTB2-Ch-5a-PSoC-6-Core.pdf"
I am using my CY8CKIT-062-WiFi-BT instead of CY8CKIT-062S-43012 but I don't see the message "I feel good!" on the TFT display.  It just displays WHITE panel.

When compared with an example on https://github.com/cypresssemiconductorco/display-tft-st7789v#quick-start it was found that there is no initialization of the TFT controller.  The message will appear when

1) Add an additional include of "cy8ckit_028_tft.h"

```
#include "cy8ckit_028_tft.h"
```

2) Initialize TFT with "cy8ckit_028_tft_init(0,0,0,0);" prior to call the "GUI_Init();"

```
    __enable_irq();

    result = cy8ckit_028_tft_init(0,0,0,0);
    CY_ASSERT(result == CY_RSLT_SUCCESS);

    GUI_Init();
```

**Steps to Reproduce:**

1. Follow Exercise 8 with CY8CKIT-062-Wifi-BT

2.

3.

**Expected Result:**

A message "I feel good!" on the DIsplay

**Actual Result:**

No message on the TFT Display.
All pixels are WHITE.

**Frequency (1/1, 1/50, 1/100):**

1/1

**Build/Commit:**

Please specify the release, like 1.4.1. If there is no release, give us the commit for the code you used.
V1.4

**Target:**

The chip or kit you are working with
CY8CKIT-062-Wifi-BT and CY8CKIT-028-TFT

**Host OS and Version:**

Windows

**Compiler:**

Include the version!
Modus ToolBox V2.1

**Environment**

Provide the context surrounding this problem. Perhaps an IDE, a debug probe, an RTOS, etc. (and versions)

**Comment by rmbk19** [comment]
Hi @noritan, 

Thanks again! This will also be fixed in the next release tentatively by next week.

Regards, 
Rakshith

**Comment by rmbk19** [comment]
Fixed in version 1.4.1
